UX-Guide-Metadata/draft/techniques/onix-metadata/index.html

```diff
@@ -580,9 +580,49 @@ <h4>Instructions</h4>
 				<p>This technique relates to <a href="https://www.w3.org/2021/09/UX-Guide-metadata-2.0/principles/#accessibility-summary">Accessibility summary key information</a>.</p>
 
+				<p>This algorithm takes the <var>onix_record_as_text</var> argument: a UTF-8 string representing the ONIX record.</p>
+
+				<p class="note">
+					Being human-written, the accessibility summary and addendum will appear in the original language of the publication. Therefore it is necessary to take care of setting up the correct language tag information.
+				</p>
+
+				<p class="note">Accessibility summary existed when no system exposed the computed accessibility metadata. With more and more systems displaying this information, most of the accessibility summary may be redundant. It may be advised to display only the accessibility addendum. </p>
+
+				<h4>Understanding the variables</h4>
+				<dl>
+					<dt><var>accessibility_summary</var></dt>
+					<dd>
+						<p>Returns the value of <a href="https://ns.editeur.org/onix/en/196/00">code 00 of codelist 1196</a>  (Accessibility summary) if present in the ONIX record, otherwise if false it means that the metadata is not present.</p>
+						<p>This means there is a human-written text containing a short explanatory summary of the accessibility of the product or the URL of a web page comprising such a summary. Summarizes the already existent information and may add information that the publisher could not express with the other codes.</p>
+					</dd>
+					<dt><var>accessibility_addendum</var></dt>
+					<dd>
+						<p>Returns the value of <a href="https://ns.editeur.org/onix/en/196/92">code 92 of codelist 196</a> (Accessibility addendum) if present in the ONIX record, otherwise if false it means that the metadata is not present.</p>
+						<p>This means there is a human-written text containing a short addendum to the accessibility detail of the product. Contains precisions that the publisher could not express with the other's codes.</p>
+					</dd>
+					<dt><var>language_of_text</var></dt>
+					<dd>
+						<p>Returns the value of nearest  <a href="https://ns.editeur.org/onix/en/22/01">code 01 of codelist 22</a> Language of text)  if present in the ONIX record, otherwise if false it means that the metadata is not present.</p>
+						<p>This indicates the main language of the content and therefore the most probable language of accessibility summary and addendum.</p>
+					</dd>
+				</dl>
+				<h4>Variables setup</h4>
+				<ol class="condition">
+					{{/*  <li><b>LET</b> <var>onix</var> be the result of calling <a href="#pre-processing">pre processing</a> given <var>onix_record_as_text</var>.</li>  */}}
+				</ol>
+				<h4>Instructions</h4>
+				<ol class="condition">
+					<li><span><b>IF</b> <var>accessibility_addendum</var><b>AND</b> <var>accessibility_summary</var>
+						<b>THEN</b> display the content of <var>accessibility_addendum</var> in a Span with language_of_text as an argument</li>
```

Comment:
I would not use "Span" because it is only related to HTML, I would write "display the content of accessibility_addendum with language_of_text as language"